storage/src/archive/benches/utils.rs

```diff
@@ -57,3 +61,29 @@ pub async fn append_random(archive: &mut ArchiveType, count: u64) -> Vec<Key> {
     archive.sync().await.unwrap();
     keys
 }
+
+pub(super) fn new_bench_runtime_config() -> TokioConfig {
```

Comment:
I suspect we could put this logic into `runtime`. I think we could add a new init function for `tokio::Config` (like we do for `deterministic::Config::timed`) like `env_storage` (or something catchier).

Reply:
https://github.com/commonwarexyz/monorepo/blob/2af26fde06b07c7e1f2df67545cc61603a78499f/runtime/src/deterministic.rs#L436-L444

(It would probably make sense to use a `Builder` pattern here so you can't just change one custom thing).